docs/ops/detection/ExperimentalDetectronTopKROIs_6.md

```diff
@@ -0,0 +1,56 @@
+## ExperimentalDetectronTopKROIs <a name="ExperimentalDetectronTopKROIs"></a> {#openvino_docs_ops_detection_ExperimentalDetectronTopKROIs_6}
+
+**Versioned name**: *ExperimentalDetectronTopKROIs-6*
+
+**Category**: Object detection, Sort
+
+**Short description**: An operation *ExperimentalDetectronTopKROIs* is TopK operation applied to probabilities of input ROIs
+
+**Detailed description**: Operation performs probabilities sorting for input ROIs and returns *max_rois* number of ROIs.
+
+**Attributes**:
+
+* *max_rois*
+
+    * **Description**: *max_rois* attribute specifies maximal numbers of output ROIs.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 0
```

Comment:
It is an optional attribute (see https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/vpu/graph_transformer/src/stages/exp_topkrois.cpp#L65).

Reply:
Ok, updated.